### PR TITLE
Add Redux lesson

### PR DESCRIPTION
**Pull request by PavelGavlik**

### CODE REVIEW

File: redux/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Managing application state - redux
+
+## Agenda
+
+1. Problem introduction
+  - Components are [too long](https://github.com/msd-code-academy/06-cinema-portal-redux/blob/master/src/PurchasePage.js)
```
Comment (PavelVanecek) [inline redux/README.md:6]: I would link to specific commit hash as the `master` branch may diverge

Comment (PavelGavlik) [reply redux/README.md:6]: Good thinking, but we want to change master a bit and this file will contain some new TODOs for participants.

**Review by PavelVanecek — APPROVED** [review]

File: redux/README.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  - We need pass props (data or event handlers) up and down
+  - We don't have enough abstractions for app to grow
+
+1. Time-travel debugging in Redux devtools
```
Comment (Aprillion) [inline redux/README.md:10]: does not render as ordered list on https://github.com/msd-code-academy/lessons/blob/10153ba9a230646aa6ff8f9fe5b8c693777e4230/redux/README.md

**edit**: works fine on https://github.com/msd-code-academy/lessons/blob/master/redux/README.md

File: redux/README.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  - Clone https://github.com/zalmoxisus/redux-devtools-extension/tree/master/examples
+  - Go to `examples/counter/`
+  - Install [Redux devtools extension](https://chrome.google.com/webstore/detail/redux-devtools/lmhkpmbekcpmknklioeibfkpmmfibljd)
+  - Open example and explore Redux devtools
+
+1. Redux concepts
```
Comment (Aprillion) [inline redux/README.md]: 3.

File: redux/README.md (hunk 1 above)
Comment (Aprillion) [inline redux/README.md]: as long as it's merged before the lesson...

**Pull request merged**